### PR TITLE
Assist on Wear OS: Fix procedure on making Assist the default

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Assist on Wear OS: 
- Fix procedure on making Assist the default
- add procedure on assigning Assist to a key

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/voice_control/android.markdown [hunk 1]
```diff
@@ -118,8 +118,19 @@ To define Home Assistant Assist as default assistant app, follow these steps:
...
+
+### Assigning a button to Assist
+
+Depending on your watch, you can assign Assist to a button so that you can start it directly with a long or double press.
+
+1. On your watch, go to **Settings** > **Advances features** > **Customize keys**.
+2. Assign a key:
+   - To use double press, tap **Home key** > **Double press**. From the list of apps, select **HA: Assist**.
+   - To use long press, tap **Home key**, then tap **press and hold**. Select **Assistant**. 
+     - Then long press the home key, and from the selection, select **HA: Assist**.
```
Comment (jpelgrom) [inline source/voice_control/android.markdown:132]: To make it more confusing: choosing HA on long press here is (I think?) Galaxy Watch specific behavior, and only happens if you have also set Home Assistant as the default digital assistant app.

Comment (c0ffeeca7) [reply source/voice_control/android.markdown:132]: Thank you, @jpelgrom! Added a conditional.

**Review by frenck — CHANGES_REQUESTED** [review]
LGTM, thanks @c0ffeeca7 👍 

../Frenck

**Review by frenck — APPROVED** [review]
Darn it, wrong button again. Can't get used to these new hotkeys my browser plugin has :(

../Frenck

**Pull request merged**